src/checkbox/styles.scss

```diff
@@ -22,7 +22,7 @@ div.root {
 .input {
```

Comment:
Can this class be moved from checkbox, radio and toggle to the abstract-switch itself?

Reply:
I agree that this seems repetitive, but If we put this in the abstract switch I think the CSS selector would become more fragile since we need the scope of the component to establish the dynamic class name (e.g.  styles.input).  We would have to rely on a static class or strictly on element selectors.

I think the CSS to establish the focus needs to be scoped to the component that "owns" the HTML that is delivering the style.

Reply:
Ok, we can handle that as a follow-up.